server/common/utils/userUtils.js

```diff
@@ -29,7 +34,36 @@ const extractUserProfile = (profile, accessToken) => {
   };
 };
 
+const removeDemoUserData = id =>
+  List.find(
+    {
+      $or: [{ ownerIds: id }, { memberIds: id }, { viewersIds: id }]
```

Comment:
should we check all ids here ?? 
not only ownerIds??

Reply:
We should check ids in one array, but only in viewersIds. The demo user gets few demo lists, sometimes they are an owner, sometimes a member, sometimes a viewer. We want to remove all list where demo user is a viewer.